wormhole-connect/src/utils/wallet/solana.ts

```diff
@@ -196,24 +239,23 @@ export async function signAndSendTransaction(
   return signature;
 }
 
-// this will throw if the simulation fails
+// This will throw if the simulation fails
 async function createPriorityFeeInstructions(
   connection: Connection,
   transaction: Transaction | VersionedTransaction,
   commitment?: Commitment,
 ) {
   let unitsUsed = 200_000;
   let simulationAttempts = 0;
+  let simulationAttemptsForKnownErrors = 0;
 
   simulationLoop: while (simulationAttempts < 5) {
```

Comment:
I don't think this check makes sense anymore, we could change it to `while (true)`. We will break out of the loop when we've exhausted known or unknown # of retries (exception gets thrown) or the simulation succeeds.

Reply:
Yup.